datasets/us/bis_denied/crawler.py

```diff
@@ -37,7 +37,7 @@ def parse_row(context: Context, row):
     sanction = h.make_sanction(context, entity, key=citation, program_key=PROGRAM_KEY)
     sanction.add("program", citation)
     h.apply_date(sanction, "startDate", row.pop("effective_date"))
-    h.apply_date(sanction, "endDate", row.pop("expiration_date"))
+    h.apply_date(sanction, "endDate", row.pop("expiratin_date"))
```

Comment:
ermahgerd

Just a suggestion to save us changing this back next week: Maybe we can try the proper spelling, and if that's unsuccessful, try the redneck spelling